tutorials/introductory/usage.py

```diff
@@ -189,9 +190,9 @@
 # Making a helper functions
 # -------------------------
 #
-# If you need to make the same plots over and over again with different data sets,
-# or want to easily wrap Matplotlib methods, use the recommended signature function
-# below.
+# If you need to make the same plots over and over again with different data
+# sets, or want to easily wrap Matplotlib methods, use the recommended
```

Comment:
was this not wrapped correctly before?

Reply:
It was 82 chars long and I got a flake8 warning (you wrapped it later in 26fc6a1bb9)